caas/kubernetes/provider/k8s_test.go

```diff
@@ -768,19 +768,17 @@ func (s *K8sBrokerSuite) TestCreate(c *gc.C) {
 			Name:   "test",
 		},
 	})
-	gomock.InOrder(
-		s.mockNamespaces.EXPECT().Create(gomock.Any(), ns, v1.CreateOptions{}).
-			Return(ns, nil),
-	)
+	s.mockNamespaces.EXPECT().Create(gomock.Any(), ns, v1.CreateOptions{}).
+		Return(ns, nil)
 
-	err := s.broker.Create(
-		envcontext.WithoutCredentialInvalidator(context.Background()),
+	err := s.broker.CreateModelResources(
+		context.Background(),
 		environs.CreateParams{},
 	)
 	c.Assert(err, jc.ErrorIsNil)
 }
 
-func (s *K8sBrokerSuite) TestCreateAlreadyExists(c *gc.C) {
+func (s *K8sBrokerSuite) TestValidateProviderForNewModelAlreadyExists(c *gc.C) {
```

Comment:
Hey Ian, just a suggestion:
Should we also add a TestValidateProviderForNewModelSuccess here? Since we are saying that validation is true if k8serrors.IsNotFound error is returned from Get api